assets/app/view/game/route_selector.rb

```diff
@@ -270,6 +271,52 @@ def actions(render_halts)
           h(:button, { style: submit_style, on: { click: submit } }, 'Submit ' + revenue + subsidy),
         ])
       end
+
+      def dividend_chart
+        step = @game.active_step
+        return nil unless step.respond_to?(:chart)
```

Comment:
if you return nil here does the method calling this call compact?

Reply:
Yes. It works fine if step.chart is not implemented.